fe/fe-core/src/main/java/org/apache/doris/catalog/SparkResource.java

```diff
@@ -103,19 +104,22 @@ public static DeployMode fromString(String deployMode) {
     // broker username and password
     @SerializedName(value = "brokerProperties")
     private Map<String, String> brokerProperties;
+    @SerializedName(value = "envConfigs")
+    private Map<String, String> envConfigs;
```

Comment:
Whether the `envConfigs` can be overwrited in users' load sqls.  
If so, please add the corresponding variable and update logic in `ResourceDesc`

Reply:
can be overwrite，the code is in updateProperties function